dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/ExpressionDimensionItemController.java

```diff
@@ -44,4 +46,28 @@
 @ApiVersion( { DhisApiVersion.DEFAULT, DhisApiVersion.ALL } )
 public class ExpressionDimensionItemController extends AbstractCrudController<ExpressionDimensionItem>
 {
+    private static final String PRIVATE_ACCESS = "--------";
+
+    /**
+     * virtual preCreateEntity method is used to set default public access to
+     * private (only creator can read and manipulate data)
+     *
+     * @param entity
+     * @throws Exception
+     */
+    @Override
+    protected void preCreateEntity( ExpressionDimensionItem entity )
+        throws Exception
+    {
+        Sharing sharing = SharingUtils.generateSharingFromIdentifiableObject( entity );
+
+        if ( sharing.getPublicAccess() == null )
+        {
+            sharing.setPublicAccess( PRIVATE_ACCESS );
+
+            entity.setSharing( sharing );
+        }
+
+        super.preCreateEntity( entity );
+    }
```

Comment:
This can be achieved by set `schema.setDefaultPrivate( true );` in ExpressionDimensionItemSchemaDescriptor